dags.yaml

```diff
@@ -1343,3 +1343,19 @@ bqetl_glam_export:
   tags:
     - impact/tier_2
   description: DAG to prepare GLAM data for public export.
+
+bqetl_crash:
+  schedule_interval: 0 2 * * *
+  default_args:
+    depends_on_past: false
+    email:
+      - dthorn@mozilla.com
+      - telemetry-alerts@mozilla.com
+    email_on_failure: false
```

Comment:
Was this meant to be an email on failure and no email on retry?

Reply:
yep, thanks for catching